infra/app/app-config/env-config/outputs.tf

```diff
@@ -34,6 +34,12 @@ output "service_config" {
       # For job configs that don't define a source_bucket, add the source_bucket config property
       job_name => merge({ source_bucket = local.bucket_name }, job_config)
     }
+
+    # Identity provider configuration
+    enable_identity_provider = var.enable_identity_provider
+    # Support local development against remote resources
+    auth_callback_urls = var.domain_name != null ? ["https://${var.domain_name}", "http://localhost:3000"] : ["http://localhost:3000"]
+    logout_urls        = var.domain_name != null ? ["https://${var.domain_name}", "http://localhost:3000"] : ["http://localhost:3000"]
```

Comment:
> question 1: any way to avoid hardcoding the port? or should we add a comment explaining this?

I don't think there's a way to avoid the port.

> question 2: any security issues with enabling this callback url in prod?

Good question. It's not ideal. What do you think is the best way to exclude it for prod?

Reply:
> What do you think is the best way to exclude it for prod?

Right now, we don't have any conditionals based on the name of an environment.

Reply:
If it's only intended for some environments but not all, then I'd make a variable to env-config module that can be specified by dev.tf/staging.tf/prod.tf. Haven't thought too deeply about it but one example could be something like

```
# variables.tf
variable "extra_identity_provider_callback_url" ...
```

and then outputs could merge the two like

```
# outputs.tf
auth_callback_urls = concat([https://${var.domain_name}], [var.extra_identity_provider_callback_url])
```

Reply:
Let me know if the latest version addresses your concern.